Newsweek clone site

sinansevgi opened the pull request:
We used Bootstrap framework to create replica of the Newsweek.com website.

-Bootstrap v4.5.2 used on the project.
-We tried to use bootstrap classes wherever possible.

sinansevgi committed:
Initial Commit.

sinansevgi committed:
Header part started.

cyonii committed:
Add .gitignore file

cyonii committed:
Remove obsolete image

cyonii committed:
Complete header top content

added genericon.css for icons

sinansevgi committed:
Navbar menu added.

sinansevgi committed:
Middle section added to main page.

-Top story section added.
-My turn section added.
-More stories section added.

sinansevgi committed:
Body structure changed.

cyonii committed:
Add contents on right column

sinansevgi committed:
Editor's Pick and In the magazine sections added.

cyonii committed:
Added 'news by category' section

sinansevgi committed:
Images replaced with picsum service

cyonii committed:
Add subscribe and newsletter

add some resembling effects on the origin website like hover effect

sinansevgi committed:
Footer part added.

-Footer created.
-Image labels added.
-Footer fonts fixed.

sinansevgi committed:
Update README.md

sinansevgi committed:
Merge branch 'newsweek-clone-site' of https://github.com/sinansevgi/newsweek-clone into newsweek-clone-site

sinansevgi committed:
Screenshot added.

-Linter configuration for github added.
-Linter errors fixed.

sinansevgi committed:
Screenshot file name error fixed.

sinansevgi committed:
AccessLint accessibility suggestions applied.

sinansevgi committed:
rel attribute for links updated.

umairarshadbutt commented on css/style.css:
```diff
@@ -0,0 +1,213 @@
+}
+
+.text-dark {
+  color: #000 !important;
```
Remove `!important`

umairarshadbutt commented on css/style.css:
```diff
@@ -0,0 +1,213 @@
+}
+
+.text-blunt {
+  color: #494f54 !important;
```
Remove `!important`

umairarshadbutt commented on css/style.css:
```diff
@@ -0,0 +1,213 @@
+}
+
+.border-thick {
+  border-width: 4px !important;
```
Remove `!important`

umairarshadbutt reviewed (changes requested):
# Status: REQUIRED CHANGES :warning: :construction: 

Hi 
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [x] Remove `!important `from css file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)
- [x] Remove horizontal scrollbar(I'm attaching the [video](https://www.awesomescreenshot.com/video/1293309?key=f99eff64e084a95241c790d05231f407))
**Main Page**
- [x] Text should overflow on images in medium and small screens

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://sinansevgi.github.io/newsweek-clone/)|
|:---:|:---:|
|![source-page](https://user-images.githubusercontent.com/29689273/95573735-56c98280-0a45-11eb-8ad1-6fabc1c9c7ca.png)|![clone](https://user-images.githubusercontent.com/29689273/95573739-592bdc80-0a45-11eb-8698-3473585da476.png)|

**Footer**
- [x] Design responsive footer for medium and small screens

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://sinansevgi.github.io/newsweek-clone/)|
|:---:|:---:|
|![source-page](https://user-images.githubusercontent.com/29689273/95573922-9e500e80-0a45-11eb-9b4e-e722010172d1.png)|![clone](https://user-images.githubusercontent.com/29689273/95573928-a019d200-0a45-11eb-8f62-6979d0e7c735.png)|

- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

sinansevgi committed:
Footer made more responsive.

-Removed all !important markers.
-Requested bug added to subscribe section
-Overflow-x set to hidden

sinansevgi committed:
Remove unnecessary whitespace from CSS

Rhelli reviewed (approved):
# Approved! 🚀✅

Well done on implementing the changes suggested by the previous TSE. You've managed to make an exceptionally responsive website that stays true to Newsweek's own website. I have a couple of suggestions that you could implement or carry through to your next project to take your work to the next level:
 - **[Optional]** - Make sure you use the `footer` tag on your webpage footers in the same way you've used the `nav` tag. This keeps your HTML semantic and accessible.
 - **[Optional]** - Your search bar magnifying glass icon is broken on full screen due to having a property of `right: 0;`. If you remove this it should place it correctly.

If you have any questions, feel free to either @ me here on Giithub or get in contact with me via Slack!

All the best,

Rory 💂🏽‍♂️
